code/src/components/Header.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import moment from "moment";
+
+const Header = () => {
+    const date = new Date()
+
+return(
+    <header>
+     <h1>to-do list</h1>
+     <p>{moment(date).format('dddd')}</p>
```

Comment:
I was stuck on how to use moment 😅. It's nice to see how you applied it here. Much better way than using new Date() in Javascript